diff --git a/node_modules/react-native-percentage-circle/index.js b/node_modules/react-native-percentage-circle/index.js
index 2b718ee..be7f7d7 100644
--- a/node_modules/react-native-percentage-circle/index.js
+++ b/node_modules/react-native-percentage-circle/index.js
@@ -50,16 +50,16 @@ const styles = StyleSheet.create({
 });
 
 class PercentageCircle extends Component {
-  propTypes: {
-    color: React.PropTypes.string,
-    bgcolor: React.PropTypes.string,
-    innerColor: React.PropTypes.string,
-    radius: React.PropTypes.number,
-    percent: React.PropTypes.number,
-    borderWidth: React.Proptypes.number,
-    textStyle: React.Proptypes.array,
-    disabled: React.PropTypes.bool,
-  }
+  // propTypes: {
+  //   color: React.PropTypes.string,
+  //   bgcolor: React.PropTypes.string,
+  //   innerColor: React.PropTypes.string,
+  //   radius: React.PropTypes.number,
+  //   percent: React.PropTypes.number,
+  //   borderWidth: React.Proptypes.number,
+  //   textStyle: React.Proptypes.array,
+  //   disabled: React.PropTypes.bool,
+  // }
 
   constructor(props) {
     super(props);
